src/formatDate.js

```diff
@@ -8,7 +8,52 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
+  const toSeparator = toFormat.pop();
```

Comment:
The code assumes that the separator is always the last element in the format arrays. Ensure that the format arrays are structured as expected, or add validation to handle unexpected structures.